ibtk/src/lagrangian/FEProjector.cpp

```diff
@@ -322,11 +322,24 @@ FEProjector::buildLumpedL2ProjectionSolver(const std::string& system_name)
             const Elem* const elem = *el_it;
 #if NDEBUG && LIBMESH_VERSION_LESS_THAN(1, 5, 0)
             {
+                auto fe_order = fe_type.order;
+                TBOX_ASSERT(elem->default_order() == fe_order);
                 auto elem_type = elem->elem_type();
-                TBOX_ASSERT(((fe_type.order == FIRST) && ((dim == 2 && (elem_type == TRI3 || elem_type == QUAD4)) ||
-                                                          (dim == 3 && (elem_type == TET4 || elem_type == HEX8)))) ||
-                            ((fe_type.order == SECOND) && ((dim == 2 && (elem_type == TRI6 || elem_type == QUAD9)) ||
-                                                           (dim == 3 && (elem_type == TET10 || elem_type == HEX27)))));
+                switch (fe_order)
+                {
+                case FIRST:
+                    TBOX_ASSERT((dim == 1 && (elem_type == EDGE2)) ||
```

Comment:
I guess I can simplify this and just list out the `FIRST` and `SECOND` order elements that are supported, and not worry about `dim`.

I will do that now...